CONTRIBUTING_MODELS.md

```diff
@@ -1,57 +1,104 @@
+- [ ] Add `xx/xx_<task>.py` for adding task models (e.g., classifier, masked LM)  
+      [Example](https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/distil_bert/distil_bert_classifier.py)
 
-### Step 5: PR #4 and Beyond - Add XX Tasks and Preprocessors
+- [ ] Add `xx/xx_<task>_preprocessor.py` for preprocessing inputs to the task 
+      model  
+      [Example](https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/distil_bert/distil_bert_preprocessor.py)
 
-This PR is optional.
+- [ ] Add unit tests: `xx/xx_<task>_test.py` and `xx/xx_<task>_preprocessor_test.py`  
+      [Example 1](https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/distil_bert/distil_bert_classifier_test.py),  
+      [Example 2](https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/distil_bert/distil_bert_preprocessor_test.py)
 
-- [ ] An `xx/xx_<task>.py` file for adding a task model like classifier, masked LM, etc. \[[Example](https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/distil_bert/distil_bert_classifier.py)\]
-- [ ] An `xx/xx_<task>_preprocessor.py` file which has the preprocessor and can be used to get inputs suitable for the task model \[[Example](https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/distil_bert/distil_bert_preprocessor.py)\].
-- [ ] `xx/xx_<task>_test.py` file and `xx/xx_<task>_preprocessor_test.py` files which have unit tests for the above two modules \[[Example 1](https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/distil_bert/distil_bert_classifier_test.py) and [Example 2](https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/distil_bert/distil_bert_preprocessor_test.py)\].
-- [ ] A Colab notebook link in the PR description, demonstrating that the output of the preprocessor matches the output of the original preprocessor \[[Example](https://colab.research.google.com/drive/1GFFC7Y1I_2PtYlWDToqKvzYhHWv1b3nC?usp=sharing)].
+- [ ] Colab notebook link in the PR description to validate that the 
+      preprocessor output matches the original  
+      [Example](https://colab.research.google.com/drive/1GFFC7Y1I_2PtYlWDToqKvzYhHWv1b3nC?usp=sharing)
+
+- [ ] Add a Colab notebook demonstrating end-to-end usage of the task model, 
+      showing matching outputs and a fine-tuning demo
+
+### Step 6: PR #4 and Beyond - Add `XXPresets`, Weights, and End-to-End Validation
+
+- [ ] Add `xx/xx_presets.py` with links to weights uploaded to Kaggle KerasHub  
+      [Example](https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/distil_bert/distil_bert_presets.py)
+
+- [ ] Stage the model presets on KerasHub’s [Kaggle org page](https://www.kaggle.com/organizations/kerashub) using this [invite link](https://kaggle.com/organizations/kerashub/invite/c4b8baa532b8436e8df8f1ed641b9cb5)
```

Comment:
We might not want to make this invite link public. Won't anyone be able to join the org with this? What kind of permissions does this get you (model creation? model deletion?).